cosmwasm/contracts/global-accountant/src/contract.rs

```diff
@@ -378,7 +389,22 @@ fn handle_governance_vaa(
                 .add_attribute("chain", chain.to_string())
                 .add_attribute("emitter_address", emitter_address.to_string()))
         }
-        Action::ModifyBalance {
+        _ => bail!("unsupported governance action"),
+    }
+}
+
+fn handle_accountant_moduleernance_vaa(
+    deps: DepsMut<WormholeQuery>,
+    info: &MessageInfo,
+    body: Body<accountant_module::GovernancePacket>,
+) -> anyhow::Result<Event> {
+    ensure!(
+        body.payload.chain == Chain::Any || body.payload.chain == Chain::Wormchain,
+        "this governance VAA is for another chain"
```

Comment:
It would be nice if this error message were distinct from `handle_token_governance_vaa` when it checks its governance target chain.